src/cpl.rs

```diff
@@ -57,6 +59,20 @@ impl CslStringList {
         Ok(())
     }
 
+    /// Adds the string `value` to the list.
+    ///
+    /// Returns `Ok<()>` on success, `Err<GdalError>` if `value` cannot be converted to a C string.
+    ///
+    /// See: [`CSLAddString`](https://gdal.org/api/cpl.html#_CPPv412CSLAddStringPPcPKc)
+
+    pub fn add_string(&mut self, value: &str) -> Result<()> {
+        let v = CString::new(value)?;
+        unsafe {
+            self.list_ptr = CSLAddString(self.list_ptr, v.as_ptr());
```

Comment:
Please move the assignment out of the `unsafe` block (move it after the `=`), it's easier to see what exactly is unsafe that way.

And if it's not too hard, can you squash at the end?

Reply:
Done and done!